sentry/src/main/java/io/sentry/IHub.java

```diff
@@ -402,14 +403,14 @@ ITransaction startTransaction(
   SentryTraceHeader traceHeaders();
 
   /**
-   * Associates {@link ISpan} with the {@link Throwable}. Used to determine in which trace the
-   * exception has been thrown in framework integrations.
+   * Associates {@link ISpan} and the transaction name with the {@link Throwable}. Used to determine
+   * in which trace the exception has been thrown in framework integrations.
    *
    * @param throwable the throwable
-   * @param span the span context
+   * @param span a pair of span context and transaction name
    */
   @ApiStatus.Internal
-  void setSpanContext(@NotNull Throwable throwable, @NotNull ISpan span);
+  void setSpanContext(@NotNull Throwable throwable, @NotNull Pair<ISpan, String> span);
```

Comment:
Perhaps we can come up with a dedicated class, instead of `Pair`. I am open for suggestions as I could not figure out a class name that would add value.

Reply:
Pair or Tuple are fine